packages/node-cli/src/core/CommandsMetadataService.js

```diff
@@ -8,7 +8,7 @@ const path = require('path');
 const FileUtils = require('../utils/FileUtils');
 const { SDK_COMMANDS_METADATA_FILE, NODE_COMMANDS_METADATA_FILE, COMMAND_GENERATORS_METADATA_FILE } = require('../ApplicationConstants');
 const SDK_WRAPPER_GENERATOR = 'commands/SdkWrapperCommandGenerator';
-let COMMANDS_METADATA_CACHE;
+let COMMANDS_METADATA_CACHE = null;
```

Comment:
Shouldn't leaving this undefined work the same way?

Reply:
Yes, but I (personally) prefer to initialize variables and not leave them undefined. However if you prefer undefined, I can change it, the code will work the same way

Reply:
Just wanted to understand the change. I don't have a problem with initializing it to null.

Reply:
This is an interesting conversation and I think we should talk with all the team for making a decission to make our code consistent. Maybe even make it one of our agreements.
I'm not sure but I also think that making one decission or another could lead to some implications all of us should be aware of.